slink-main/smartrecruiters.js

```diff
@@ -24,6 +24,9 @@ const getApplicants = async ({
   return R.flatten(await batchedApplicants);
 };
 
+// specialized function to extract candidates, who have substatus set to 'ONBOARDING'
+const getApplicantsOnboarding = async () => getApplicants({ subStatus: 'ONBOARDING' });
```

Comment:
We'll need to find a way to merge in the date filter here, too.  You'll have some conflicts, unfortunately when you pull my code.  I'm going to try to stay out of this code for the near term to prevent more conflicts if possible.